python/delta_sharing/tests/test_delta_sharing.py

```diff
@@ -364,20 +394,31 @@ def list_all_tables(
 def test_load_as_pandas_success(
     profile_path: str,
     fragments: str,
+    jsonPredicateHints: Optional[Dict[str, Any]],
+    predicateHints: Optional[Sequence[str]],
     limit: Optional[int],
     version: Optional[int],
     expected: pd.DataFrame
 ):
-    pdf = load_as_pandas(f"{profile_path}#{fragments}", limit, version, None)
+    pdf = load_as_pandas(
+        f"{profile_path}#{fragments}",
+        jsonPredicateHints,
+        predicateHints,
+        limit,
+        version,
+        None
+    )
     pd.testing.assert_frame_equal(pdf, expected)
 
 
 @pytest.mark.skipif(not ENABLE_INTEGRATION, reason=SKIP_MESSAGE)
 @pytest.mark.parametrize(
-    "fragments,version,timestamp,error",
+    "fragments,jsonPredicateHints,predicateHints,version,timestamp,error",
```

Comment:
Is there a unit test that specifies these predicate hints and verifies that they are used correctly?

Reply:
@chakankardb 
Thank you for the comment, and yes, I have to add an unit test to check if these new parameters are used correctly. I will ping you again when I add the unit test.